docs/api/go/getting-started.md

```diff
@@ -0,0 +1,109 @@
+# Get started
+
+In this guide we will demonstrate how to:
+
+1. Connect to the zkSync network.
+2. Deposit assets from Ethereum into zkSync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of go and you have it configured on your local machine.
+Also, the required version is >= 1.17 and Go modules were used.
+
+
+## Installation
+Add this package to your project:
+
+```go 
+
+get github.com/zksync-sdk/zksync2-go
+
+```
+
+## How to init main components
+Using these three instances `ZkSync Provider`, `EthereumProvider` and `Wallet`, you can perform all basic actions with ZkSync network, just explore their public methods and package helpers.
+
+A few examples are below:
+
+## Depositing funds
+
+```go
+
```

Comment:
I'm not familiar with Go but, should we have to include some imports on the top of this script?